Makefile

```diff
@@ -133,6 +133,7 @@ deploy-s3:
 	nht deploy-hashed-assets --directory public/n-ui --monitor-assets
 
 deploy: deploy-s3 npm-publish rebuild-user-facing-apps
+	@$(MAKE) change-api
```

Comment:
is there a particular reason to do it this way instead of as another dependency of `deploy` (i.e. after `rebuild-user-facing-apps`?)

Reply:
None! Hadn't thought about using it as a target dependency. Will update it to that.